### DISCUSSION TITLE
no distributor in EU?

### DISCUSSION THREAD

**Discussion by mi-hol** [JGAguado/Smart_Powermeter#29] [General]
Great product but shipping from US to EU is expensive and VAT and import charges are to be added on top.
Therefore I wonder if a distributor in EU has it available?

![image](https://github.com/JGAguado/Smart_Powermeter/assets/22799428/1366bcc2-5677-4bbd-ac92-ae0d04808920)

**Comment by JGAguado** [comment]
Thanks for the support! (and that typo fix 😉 ) 
I started shipping from EU (from Austria) but the shipment costs were the same (or even more expensive) within EU, and even worst when shipping out of EU. 

![image](https://github.com/JGAguado/Smart_Powermeter/assets/7847822/6f689375-7cd6-417b-8c1c-286bb910c32f)

However, if any company (that may achieve better shipment rates) would eventually decide to approach me in order to become a EU distributor, I'm open to talk 😃

**Reply by mi-hol** [reply]
https://www.myhermes.at/privatkunden/produkte-preise/preise-fuer-den-versand/
![image](https://github.com/JGAguado/Smart_Powermeter/assets/22799428/77f72d13-1c33-4e92-b1ca-ba9623c219e7)

via buying 10 shipment vouchers in advance costs come down a bit further.
DPD, GLS and other offer shipments for businesses cheaper too
